content/en/getting_started/integrations/aws.md

```diff
@@ -0,0 +1,250 @@
+
+Once the Agent is installed, it's graphically represented within the [Infrastructure List][18] with a bone icon:
+{{< img src="getting_started/integrations/infrastructure-list.png" alt="An image from the infrastructure list of the Datadog account. Two hosts are displayed in a list format. Both hosts show the AWS icon for the AWS integration and 'aws' shown in a blue box to show they are associated with the AWS integration. One host also shows a dog-bone icon and blue boxes for 'ntp' and 'system'.">}}
+
+As can be seen in the screenshot above, the host with the Datadog Agent is reporting data from the [System][19] and [NTP][20] checks. The System check provides metrics around CPU, memory, filesystem, and I/O, providing additional insights into the host. You can enable additional [integrations][21] to suit the environment and use case, and [DogStatsD][22] can additionally be leveraged for sending custom metrics directly to Datadog. 
+
+See the [FAQ on why you should install the Datadog Agent on your cloud instances][23] for more information about the benefits of this approach.
+
+### Using the Datadog Agent with Amazon Container Services
+
+For containerized environments, you can use the Datadog Agent, whether you're managing your instances or leveraging [Fargate][24] for a serverless environment.
+
+#### ECS with EC2 launch type
+
+Use the [Amazon ECS documentation][25]  to run the [Datadog Docker Agent][26] on the EC2 instances in your ECS cluster. Review the [Amazon ECS Data Collection documentation][27]  to see the metrics and events reported to your Datadog account.
+
+#### ECS with Fargate launch type
+
+Use the [Amazon ECS on AWS Fargate documentation][28]  to run the Agent as a container in the same task definition as your application. **Note**: Datadog Agent version 6.1.1 or higher is needed to take full advantage of the Fargate integration.
+
+#### EKS
+
+You don't need any specific configuration for Amazon Elastic Kubernetes Service (EKS), as mentioned in the [Kubernetes Distributions documentation][29]. Use the [dedicated Kubernetes documentation][30] to deploy the Agent in your EKS cluster.
+
+#### EKS with Fargate
+
+Since Fargate pods are managed by AWS, they exclude host-based system checks like CPU and memory. In order to collect data from your AWS Fargate pods, use the [Amazon EKS on AWS Fargate documentation][31] to run the Agent as a sidecar of your application pod with custom role-based access control (RBAC). **Note**: This requires Datadog Agent version 7.17 or higher.
```

Comment:
```suggestion
Because Fargate pods are managed by AWS, they exclude host-based system checks like CPU and memory. To collect data from your AWS Fargate pods, use the [Amazon EKS on AWS Fargate documentation][31] to run the Agent as a sidecar of your application pod with custom role-based access control (RBAC). **Note**: This requires Datadog Agent version 7.17 or higher.
```